packages/components/header/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/f36-header",
-  "version": "4.0.0-alpha.6",
+  "version": "4.0.0-alpha.7",
```

Comment:
https://www.npmjs.com/package/@contentful/f36-header

Reply:
According to `npm view @contentful/f36-header@4.0.0- time --json`, all versions have been published at the time they've been worked on, and none of them have been unpublished.

```JSON
{
  "4.0.0-alpha.1": "2023-12-08T13:24:40Z",
  "4.0.0-alpha.2": "2024-04-11T08:11:24Z",
  "4.0.0-alpha.3": "2024-04-11T08:39:00Z",
  "4.0.0-alpha.4": "2024-04-11T08:43:02Z",
  "4.0.0-alpha.5": "2024-04-11T08:47:19Z",
  "4.0.0-alpha.6": "2024-04-18T10:05:53Z",
  "4.0.0-alpha.7": "2024-04-24T09:50:13Z"
}
```